pylint/testutils/output_line.py

```diff
@@ -78,19 +83,26 @@ def from_msg(cls, msg):
         )
 
     @classmethod
-    def get_column(cls, column):
+    def get_column(cls, column: Union[int, str]) -> str:
         if not PY38_PLUS:
             return ""  # pragma: no cover
         return str(column)
 
     @classmethod
-    def from_csv(cls, row):
+    def from_csv(
+        cls, row: Union[List[str], Tuple[str, str, str, str, str], str]
+    ) -> "OutputLine":
         try:
-            confidence = row[5] if len(row) == 6 else interfaces.HIGH.name
+            if len(row) == 6:
+                confidence = row[5]  # type: ignore # mypy does not recognize that this cannot fail
+            else:
+                confidence = interfaces.HIGH.name
             column = cls.get_column(row[2])
             return cls(row[0], int(row[1]), column, row[3], row[4], confidence)
         except Exception as e:
-            raise MalformedOutputLineException(row, e) from e
+            if isinstance(e, ValueError):
+                raise MalformedOutputLineException(row, e) from e
+            raise e
```

Comment:
```suggestion
            raise
```
That would be enough, but why add the `isinstance` check in the first place?

Reply:
Because `MalformedOutputLineException` requires a `ValueError` and `mypy` complains otherwise.